src/components/global/carouselWrapper/carouselWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import React, { FC, ReactNode } from "react";
+import Slider, { Settings } from "react-slick";
+
+import * as styled from "./carouselWrapper.styles";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+export interface ICarouselSliderComp {
+  children: ReactNode;
+  settings: Settings;
+}
```

Comment:
Dit is ook wel iets anders en handig om te weten. Als ge een component maakt die args vereist, kunt ge voor die args een interface maken. In die interface zet ge alle args met types, en in de declaration van de functie geeft ge deze interface mee. (FC<ICarouselSliderComp> is 1 manier om het te doen, zijn ook nog andere manieren)